src/set_relation/set_relation.cc

```diff
@@ -2944,5 +2944,79 @@ Relation* Relation::addConstraintsDueToMonotonicity() const {
     return retval;
 }
 
+/*****************************************************************************/
+#pragma mark -
+/*************** setExpType **************************************************/
+
+void Conjunction::setExpType()
+{
+    std::list<Exp*>::iterator expIter = mEqualities.begin();
+    while (expIter != mEqualities.end()) {
+        (*expIter)->setEquality();
+        expIter++;
+    }
+    expIter=mInequalities.begin();
+    while (expIter != mInequalities.end()) {
+        (*expIter)->setInequality();
+        expIter++;
+    }
+}
+
+void SparseConstraints::setExpType()
```

Comment:
Don't need this method. Just set the expression type in Conjunction::addInequality and Conjunction::addEquality